source/users.rst

```diff
@@ -177,6 +242,7 @@ Request:
     "username": "example",
     "password": "password",
     "email": "example@example.com"
+    "roles": ["intern"],
```

Comment:
According to our documentation, when returning associated objects bundled with a main object (activities or projects for a time, for example) a list of slugs is returned. 

In practice, I believe the clients typically request all activities in a separate api call, and if needed, locally associate the display name of the activity from that data set, with its slug as listed in the time object. 

This kind of bundling essentially defines the user object as containing a field which is a list of role slugs - this is distinct from bundling associated objects (like revisions of times) where full objects are returned as children of the parent object.